src/vaishnaviVJun20/Math.java

```diff
@@ -0,0 +1,40 @@
+package vaishnaviVJun20;
+
+public class Math {
+    double sum(double variable1, double variable2, double variable3) {
+    	double sum = variable1+variable2+variable3;
+    	return sum;
+    }
+    double average(double variable1, double variable2, double variable3) {
+    	System.out.println("Sum of three variable is: " +sum(variable1,variable2,variable3));
+    	double average = (variable1+variable2+variable3)/3;
```

Comment:
you should say average = sum(variable1,variable2,variable3)/3; Otherwise there is no need of sum method. we asked you call sum method from average. Please correct it

Reply:
Done